spec/imsc-hrm.html

```diff
@@ -247,9 +247,164 @@ <h2>Terms and Definitions</h2>
+        <li>a second party over-provisions their <a>presentation processor</a>
+        in order to ensure a complete presentation of all <a>IMSC Document
+        Instances</a>, resulting in increased run-time resource usage, code
+        complexity, etc.; or</li>
+  
+        <li>one <a>presentation processor</a> becomes the de-facto reference to
+        determine whether the presentation of a <a>IMSC Document Instance</a>
+        will succeed, at the expense of other renderers and consistency of
+        presentation.</li>
+      </ul>
 
-    <section class="informative">
-      <h2>Overview</h2>
+      <figure id='fig-intro'>
+        <img src="figures/intro-fig-1.svg" alt="The HRM prevents incomplete presentations of IMSC Instance Documents.">
+
+        <figcaption>
+          The HRM allows authors and implementers of <a>presentation
+          processors</a> to agree on the maximum complexity of <a>IMSC Document
+          Instances</a>.
+        </figcaption>
+      </figure>
+
+      <p>As illustrated in <a href="#fig-intro"></a>, by defining a method (the
+      HRM) to compute a proxy for the complexity of an <a>IMSC Document
+      Instance</a> and specifying a complexity limit based on such proxy:</p>
+
+      <ul>
+        <li>authors can ensure that their <a>IMSC Document Instances</a> do not
+        exceed this limit;</li>
+  
+        <li>implementer can design their <a>presentation processor</a> to maximize the
+        likelihood that it will be able to render correctly all <a>IMSC Document
+        Instances</a> that do not exceed this limit; and</li>
+          
+        <li>audiences can expect subtitle and caption presentation to match
+        the authorial intention.</li>
+      </ul>
+
+    </section>
+  
+    <section>
+      <h3>Why is the HRM needed to limit complexity?</h3>
+
+      <p>The HRM supplements the syntactic and structural constraints imposed in
+      [[IMSC]] by imposing constraints on the contents of the presentation.</p>
+  
+      <p>Because of the temporal and spatial variability of subtitles and
+        captions across types of content, territories and languages, it is not
+        possible to limit the complexity of an <a>IMSC Document Instance</a> using
+        only average values.</p>
+  
+      <p class="example">An average-based constraint of 840 <a>characters</a> per
+      minute could be met in multiple ways, with different rendering
+      complexities. Contrast two potential approaches. In the first, 5
+      <a>characters</a> are presented for a fraction of a second, followed by 835
+      <a>characters</a> that are then presented for over 59 seconds. This generates a
+      high rendering complexity for the 835 <a>characters</a>, since there is only a
+      brief time available to paint them. In the second, 210 <a>characters</a> must be
+      painted every 15 seconds, giving 15 seconds to prepare for the next
+      presentation. This has a much lower rendering complexity.</p>
```

Comment:
I realised this is quite a big block of text to parse in one go.

```suggestion
      <div class="example">
        <p>An average-based constraint of 840 <a>characters</a> per
          minute could be met in multiple ways, with different rendering
          complexities. Contrast two potential approaches:</p>
        <p> In the first, 5
          <a>characters</a> are presented for a fraction of a second, followed by 835
          <a>characters</a> that are then presented for over 59 seconds. This generates a
          high rendering complexity for the 835 <a>characters</a>, since there is only a
          brief time available to paint them.</p>
        <p>In the second, 210 <a>characters</a> must be
          painted every 15 seconds, giving 15 seconds to prepare for the next
          presentation. This has a much lower rendering complexity.</p>
      </div>
```